File: one-mobile/flutter_project/lib/xrauth/constants/account_const.dart
import 'dart:io';

import 'package:package_info_plus/package_info_plus.dart';

class AccountConst {
  static const String authingTokenEndpoint = '/oidc/token';
  static const String guestCreateEndpoint = '/authing/guest';
  static const String deviceAuthEndpoint = '/authing/device/code';
  static const String deviceAuthStatusEndpoint = '/authing/device/code/status';
  static const String deviceAuthingIdTokenToAccessTokenEndpoint =
      '/oauth/oidc/token';
  static const String idTokenToCredentialsGrantType =
      'http://authing.cn/oidc/grant_type/authing_token';
  static const String idTokenToCredentialsRedirectUri =
      'https://xraccount.xrspace.net.cn/authing/authn';
  static const String authingGlobalDomain = '.us.authing.co';
  static const String authingTPFiveDomain = 'tpfive-qa.us.authing.co';

  static Future<String> redirectUri(String serverDomain) async {
    final packageInfo = await PackageInfo.fromPlatform();
    return '$serverDomain/api/v1/auth/authing/authn/${Platform.operatingSystem}/${packageInfo.packageName}';
  }
}

File: one-unity/core/development/common/game-account/Runtime/Scripts/Base/AccountConst.cs
namespace TPFive.Game.Account
{
    public static class AccountConst
    {
        public const string AuthingTokenEndpoint = "oidc/token";
        public const string GuestCreateEndpoint = "authing/guest";
        public const string DeviceAuthEndpoint = "/authing/device/code";
        public const string DeviceAuthStatusEndpoint = "/authing/device/code/status";
        public const string DeviceAuthingIdTokenToAccessTokenEndpoint = "/oauth/oidc/token";
        public const string IdTokenToCredentialsGrantType = "http://authing.cn/oidc/grant_type/authing_token";
        public const string IdTokenToCredentialsRedirectUri = "https://xraccount.xrspace.net.cn/authing/authn";
    }
}